api/handler/trigger.go

```diff
@@ -190,15 +190,18 @@ func checkingTemplateFilling(request *http.Request, trigger dto.Trigger) *api.Er
 		allStates       map[string]struct{}
 	)
 
-	eventsList, err := controller.GetTriggerEvents(database, trigger.ID, page, size, eventDefaultFrom, eventDefaultTo, allMetricRegexp, allStates)
-	if err != nil {
-		return err
+	eventsList, errResponse := controller.GetTriggerEvents(database, trigger.ID, page, size, eventDefaultFrom, eventDefaultTo, allMetricRegexp, allStates)
+	if errResponse != nil {
+		return errResponse
 	}
 
-	if err := trigger.PopulatedDescription(eventsList.List); err != nil {
+	newDescription, err := trigger.PopulatedDescription(eventsList.List)
+	if errResponse != nil {
```

Comment:
```suggestion
	if err != nil {
```
А не так должно быть?

Reply:
Как бы да, но там тип не `error`, а `*api.ErrorResponse`

Reply:
Так что нельзя называть так же, как переменную `err` строчкой ниже

Reply:
Я про то, что мы как будто прочекали не ту ошибку. Вместо той, что отдал `PopulateDescription` мы сейчас чекаем errResponse

Reply:
Ааа, ты прав, я просмотрел

Reply:
Починил